src/ngraph/runtime/gpu/gpu_memory_manager.cpp

```diff
@@ -80,6 +80,7 @@ void runtime::gpu::GPUMemoryManager::allocate()
             m_argspace_mem.back().ptr, m_buffered_mem.data(), m_buffer_offset);
         // add an empty node to the end of the list and zero offset
         m_argspace_mem.push_back({nullptr, 0});
+        m_buffered_mem.resize(initial_buffer_size);
```

Comment:
I don't think it's required since we maintain our own buffer offset, but we could be safe and clear the space before resize. We could also zero-initialize the space here and in the constructor.

```
        m_buffered_mem.clear();
        m_buffered_mem.resize(initial_buffer_size, 0);
```

Reply:
Yes, it's not required but could release some resource. 
Right, it will be safe to  zero-initialize.

Reply:
let me try add a test.